docs/adr/0095-external-drive-file-picker-integration.md

```
@@ -0,0 +1,278 @@
+    return _createIntent(platformUrl, accessToken, ...);
+  }
+}
+```
+
+**Production:** `ExchangeDriveTokenInteractor` calls `POST https://<platformUrl>/auth/token_exchange` with the current OIDC `id_token` and returns the drive access token.
+
+Token storage: in-memory only, held for the duration of the intent creation call. Not persisted.
+
+### Step 3 — Create Intent
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)